src/routes/Requests.tsx

```diff
@@ -0,0 +1,698 @@
+						onClick={() => {
+							setFromDate(defaultFromDate);
+							setToDate(defaultToDate);
+							setGridFilters({
+								customer: null,
+								order: null,
+								region: null,
+								location: null,
+								yard: null,
+							});
+							setQuickSearch("");
+						}}
+					/>
+				</div>
+				{/* Grid */}
+				<div className="flex h-full flex-col space-y-4">
+					<div className="ag-theme-alpine h-full w-full grow">
+						<AgGridReact
+							ref={gridRef}
+							rowData={filteredRequests}
+							gridOptions={gridOptions}
+							rowHeight={30}
+							headerHeight={30}
+							rowSelection={"single"}
+							animateRows={false}
+							suppressDragLeaveHidesColumns={true}
+							quickFilterText={quickSearch}
+							onCellClicked={async (event) => {
+								if (event?.node?.isSelected()) {
+									await onSelectEvent(event);
+								}
+							}}
+							onCellKeyDown={async (event) => {
+								if ((event.event as KeyboardEvent).key === "Enter") {
+									await onSelectEvent(event);
+								}
+							}}
+							onRowSelected={async (event) => {
+								// Since clicking a cell also counts as selecting the row it's in, we need to make sure we're not double-firing the event
+								if (event?.node?.isSelected() && event?.event?.type === "keydown") {
+									await onSelectEvent(event);
+								}
+							}}
+							overlayNoRowsTemplate={
+								`
+                      <div className="ag-overlay" >
+                        <i className="text-sm font-medium">` +
+								`No request for ${
+									formatLocalDatetime(fromDate, "MM/dd/yyyy") +
+									(fromDate != toDate ? " - " + formatLocalDatetime(toDate, "MM/dd/yyyy") : "")
+								}` +
+								`</i>
+                      </div>`
+							}
+						/>
+					</div>
+				</div>
+			</Box>
+			<DetailsPopUp
+				isPopupOpen={isDetailPopUpOpen}
+				content={detailPopUpContent}
+				onClose={() => {
+					setIsDetailPopUpOpen(false);
+					setDetailPopUpContent(undefined);
+				}}
+			/>
+		</div>
+	);
+}
+
+/** Build the Detail for the popup from the selected data
+ * @param {RequestDisplay} request the selected object
+ * @returns {React.ReactNote} details node
+ */
+function details(request: RequestDisplay) {
+	const requestHaulers = request.requestHaulers;
+	const requestHaulerYards = new Map(
+		requestHaulers?.map((rh) => [rh?.orderProduct?.yard?.yardKey, rh?.orderProduct?.yard]),
+	);
+	const uniqueRequestHaulerYards = [...requestHaulerYards.values()];
+	console.log(uniqueRequestHaulerYards);
+	return (
+		<>
+			<h1 className="text-xl font-bold">Request Details</h1>
+			<Divider />
+			<div
+				className={twMerge(
+					"max-h-[70vh] max-w-[70vw] gap-6 text-base",
+					"grid auto-cols-[1fr_1fr_0.75fr] grid-flow-col",
+					"[&_h2]:font-medium",
+				)}>
+				<div
+					className={twMerge(
+						"grid grid-flow-row auto-rows-max grid-cols-[auto_1fr]",
+						"gap-x-2 gap-y-0.5 overflow-hidden",
+					)}>
+					<h2>Request Info:</h2>
```

Comment:
Styling isn't coming through on my end for the popups? (looks okay in Chrome, but not FireFox
![image](https://github.com/TAC-Insight/fw-scale-frontend/assets/4155618/b5c3c038-0069-443d-b736-e76c6f4c042a)

Could probably use
```suggestion
					<span className="font-semibold">Request Info:</span>
```
instead of h2's (since those are typically header stylings for starting sections/paragraphs)
![image](https://github.com/TAC-Insight/fw-scale-frontend/assets/4155618/b66d937d-05f0-4d84-86dc-987f74d465f3)

Reply:
Oh weird, the H2 should have arbitrary styling from the parent DIV since our theme doesn't style it at all by default. 🤔I can take a look at a better method for this in another issue since the Ticket Grid uses the same thing.

Reply:
https://github.com/TAC-Insight/fw-scale-frontend/issues/179